src/pages/create-index.html

```diff
@@ -118,76 +122,125 @@ <h1>
+                        </p>
 
-                    {{'query.execution.on.search' | translate}}
-                </div>
+                        <p>{{'query.must' | translate}} <span class="small-caps">SELECT</span> {{'the.following.bindings' | translate}}</p>
+                        <ul ng-show="similarityIndexInfo.isTextType() || similarityIndexInfo.isTextLiteralType()">
+                            <li><code>?documentID</code> {{'determines.id.indexed.doc' | translate}}</li>
+                            <li><code>?documentText</code> {{'determines.doc.content' | translate}}</li>
+                        </ul>
+                        <ul ng-show="similarityIndexInfo.isPredicationType()">
+                            <li><code>?subject</code></li>
+                            <li><code>?predicate</code></li>
+                            <li><code>?object</code></li>
+                        </ul>
 
-                <div class="alert alert-help mb-2" id="query-help" uib-collapse="helpHidden" ng-show="page == 3">
-                    <button class="close absolute" ng-click="toggleHelp(0)"></button>
-                    <p class="lead">
-                        {{'this.sentence.start' | translate}} <span class="small-caps">SELECT</span>
-                        {{'query.fetch.analogical.search' | translate}}
-                    </p>
+                        {{'query.execution.info' | translate}}
+                    </div>
 
-                    {{'analogical.search.info' | translate}}
+                    <div ng-if="similarityIndexInfo.isSearchTypeSelected()">
+                        <p class="lead">
+                            {{'this.sentence.start' | translate}} <span class="small-caps">SELECT</span> {{'query.fetch.on.search' | translate}}
+                        </p>
+
+                        {{'query.execution.on.search' | translate}}
+                    </div>
+
+                    <div ng-if="similarityIndexInfo.isAnalogicalTypeSelected()">
+                        <p class="lead">
+                            {{'this.sentence.start' | translate}} <span class="small-caps">SELECT</span>
+                            {{'query.fetch.analogical.search' | translate}}
+                        </p>
+
+                        {{'analogical.search.info' | translate}}
+                    </div>
                 </div>
 
                 <div class="row">
                     <div class="col-md-9">
-                        <query-editor nostorage notabs notoolbar-saved notoolbar-copy nostatus nocount norun
-                                      callback-on-change="updateDirty">
-                        </query-editor>
+                        <yasgui-component id="query-editor" yasgui-config="yasguiConfig""></yasgui-component>
+
+                        <div ng-if="similarityIndexInfo.isDataTypeSelected()">
+
+                            <div style="margin-top: 5px" ng-if="similarityIndexInfo.invalidSelectQuery"
```

Comment:
Can you replace these `style="margin-top: 5px"` with `mt-5` or similar class. See the utility classes we are using in the WB. Or just style them in some related stylesheet.